### DISCUSSION TITLE
probleme max feed

### DISCUSSION THREAD

**Discussion by Dididu12** [kachurovskiy/megaels#5] [Q&A]
BONJOUR A TOUS JAI CETTE ERREUR   (MAX FEED)   APRES AVOIR REGLER LES PARAMETRE EN HTML DE CONE TRHEAD ECT COMMENT LE RESOUDRE MERCI 
![image](https://user-images.githubusercontent.com/97460665/154846894-4b70ac29-72b6-44f4-a758-b136c9574c9b.png)

**Comment by kachurovskiy** [comment]
I believe there are 2 potential solutions for this:

1. Use less micro-stepping e.g. `400` steps per revolution instead of `800`
2. Lower the `MAX_FEED` value, e.g. if it's `40`, try `20`

Cheers!

**Reply by Dididu12** [reply]
A quelle ligne il y a tellement de ligne 😅 et que doit on modifier j'ai la vis de l'axe croisé (x,z??)qui a un pas de 1mm

**Reply by Dididu12** [reply]
Elle et déjà régler sur 20 d'origine dans le code que dois-je faire merci

**Reply by pantonvich** [reply]
I have 1mm pitch x - but have a 5/1 gear ratio - I had to use:
MOTOR_X_STEP_PER_REV 1185
SCREW_X              100
MAX_FEED             20
McSTEP_X             1

**Reply by Dididu12** [reply]
Ok merci c'est avec les systèmes a kachurovskiy

**Reply by pantonvich** [reply]
![image](https://user-images.githubusercontent.com/5418596/154859046-53bfdc0c-32ff-4ace-b361-099ded1a5c36.jpg) I have 13/77

**Reply by Dididu12** [reply]
Sa marche bien dans l'ensemble se boîtier Arduino

**Comment by pantonvich** [comment]
I had the same issue - I had to reduce both max_feed and steps. They probably should have a separate max feed for x and x other then using the same one - for my x I can still have it stall out / loose steps at faster speeds - so I need to lower mine more.

**Reply by Dididu12** [reply]
Quelle et l'axe du croisé et qu'elle et l'axe de la longueur du chariot

**Reply by pantonvich** [reply]
Sorry I tried translating but don’t understand

**Reply by pantonvich** [reply]
X and z should have been programmed to used different max feeds but unfortunately they use the same max feed value
